chore: documentation for backups in platform (#9866)
supabase/supabase: apps/reference/docs/guides/platform/backups.mdx

> ---
> id: backups
> title: Backups
> description: Available forms of backups for your Supabase project
> ---
> 
> ## Database Backups

Reviewer: style nit: I don't think we need a heading immediately following the title
![Screen Shot 2022-10-26 at 3 01 16 PM](https://user-images.githubusercontent.com/7026076/198146946-46af573b-7d68-4597-8f4c-44ed3582c316.png)

```suggestion
---
id: database-backups
title: Database Backups
description: Learn about the different ways your Supabase project's database is backed up
---
```

Database backups^ are an integral part of any disaster recovery plan. Disasters come in many shapes and sizes. It could be as simple as accidentally deleting a table column, the database crashing, or even a natural calamity wiping out the underlying hardware a database is running on. The risks and impact brought by these scenarios can never be fully eliminated, but only minimized or even mitigated. Having database backups is a form of insurance policy. They are essentially snapshots of the database at various points in time. When disaster strikes, database backups allow the project to be brought back to any of these points in time, therefore averting the crisis.

> ### Frequency of Backups

Reviewer: ```suggestion
## Frequency of Backups
```

### Restoration Process

When selecting a backup to restore to, select the closest available one made before the desired point in time to restore to. Earlier backups can always be chosen too but do consider the number of days’ worth of data that could be lost. 

```
Insert screenshot of the Dashboard with the confirmation modal 
```

The Dashboard will then prompt for a confirmation before proceeding with the restoration. The project will be inaccessible following this. As such, do ensure to allot downtime beforehand. This is dependent on the size of the database. The larger it is, the longer the downtime will be. Once the confirmation has been given, the underlying SQL of the chosen backup is then run against the project. The PostgreSQL utility [psql](https://www.postgresql.org/docs/current/app-psql.html) is used to facilitate the restoration.

```
Insert screenshot of the Dashboard of the project in restoration mode
```

As the restoration occurs, the project on the Dashboard will be inaccessible as well. The Dashboard will display a notification once the restoration completes.

```
Insert screenshot of the Dashboard of the project completing restoration
```

> 
> ---

Reviewer: style nit: I don't think we need horizontal lines (kind of signifies it should be a separate doc/page)
```suggestion
```
